Xamarin.Forms.Platform.UAP/SearchBarRenderer.cs

```diff
@@ -313,5 +313,23 @@ protected override void UpdateBackgroundColor()
 				_queryTextBox.ClearValue(Windows.UI.Xaml.Controls.Control.BackgroundProperty);
 			}
 		}
+
+		bool _isDisposed;
```

Comment:
Could we move this variable definition with the rest of the variables?https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.WPF/Renderers/SearchBarRenderer.cs#L14

Reply:
Done